Integrate pelican, falcon, and hornet AMS, add HMD to pilot helmets

red2fred2 opened the pull request:
This pull request adds HMD HUD capability to the pilot helmets when using Firewill's aircraft.
It also adds Firewill's AMS capability to the non-socom pelicans, falcons, and hornets. This is a little buggy, but would require redefining almost all the AMS scripts to fix (the scripts aren't very modular).

red2fred2 committed:
Move Pelican data out to its own folder
This will make it easier to keep pelican stuff separate from other
vheicles.

red2fred2 committed:
Add FIR AMS into non-SOCOM pelicans

red2fred2 committed:
Fix broken pylon image in pelican AMS

red2fred2 committed:
Add FIR HMD to pilot helmets
This change adds an option to use a full screen hud on firewills planes
to all pilot helmets.

red2fred2 committed:
Fix wonky bracket in pelican config

red2fred2 committed:
Add AMS to falcon
This still has the same bugs as the pelican version, but it's better
than nothing.

red2fred2 committed:
Merge pull request #1 from asullivanr/main

Pull in new changes in base repo

red2fred2 committed:
Update todo list to reflect AMS and HMD patch

red2fred2 committed:
Remove redundant pelican code
I forgot to remove this when I moved the pelican code out to its own
folder.

red2fred2 committed:
Move hornets out to their own folder

red2fred2 committed:
Fix Falcon AMS pylon picture

red2fred2 committed:
Add AMS to hornets

asullivanr committed:
Merge branch 'main' into main

asullivanr committed:
Fixed class names

Fixed class names for readability and fixed bug related to same class names.

asullivanr committed:
Fixed authorship and class string names

asullivanr commented:
[3178199](https://github.com/asullivanr/10thArmorMod/pull/6/commits/3178199effb1209c15fb414a3e99808f5b708030) was me fixing an issue that apparently would prevent a possible merge.

red2fred2 commented:
Ohh, okay. Usually that's done as part of a merge. That's why I was confused.

red2fred2 committed:
Fix falcon targeting issues
Apparently, helicopter targeting in arma gets wonky without the primary
gunner onboard. This just removes the left door gun's primary gunner
role.

asullivanr reviewed (approved):
I looked over everything and couldn't see any issues. Approving merge for the next update. I'll do any additional bug fixes then.
